docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -2373,6 +2373,72 @@
     "tab.max()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d98b298c",
+   "metadata": {},
+   "source": [
+    "### Table.idxmax()\n",
+    "\n",
+    "```\n",
+    "Table.idxmax(axis=0, skipna=True, numeric_only=False)\n",
+    "```\n",
+    "\n",
+    "Return index of first occurrence of maximum over requested axis.\n",
```

Comment:
Typo: "Over **the** requested axis"

Reply:
It's written as in the pandas documentation https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.idxmax.html
Do you think it's better to change it or keep it as it is?

Reply:
I think it comes down to preference. In my personal opinion it's more correct with the "the", but I guess if the official pandas documentation states otherwise then it's probably not entirely wrong... I'll leave it up to you since it's something so minor

Reply:
I prefer to maintain it using the Pandas API wording.